README.md

```diff
@@ -1,7 +1,36 @@
-# virtual
+# Virtual
```

Comment:
We tend to use the library name as the H1 title of readme documents. This is why it's lower case.